packages/react-components/react-tree/src/components/TreeItem/useTreeItem.tsx

```diff
@@ -239,10 +239,8 @@ export function useTreeItem_unstable(props: TreeItemProps, ref: React.Ref<HTMLDi
         role: 'treeitem',
         'aria-level': level,
         'aria-checked': selectionMode === 'multiselect' ? checked : undefined,
-        // aria-selected is required according to WAI-ARIA spec
-        // https://www.w3.org/TR/wai-aria-1.1/#treeitem
         // Casting: when selectionMode is 'single', checked is a boolean
-        'aria-selected': selectionMode === 'single' ? (checked as boolean) : 'false',
+        'aria-selected': selectionMode === 'single' ? (checked as boolean) : undefined,
```

Comment:
Hi @smhigley 
looking into this change and I cannot see any improvement with Narrator related to the [ADO issue](https://dev.azure.com/microsoftdesign/fluent-ui/_workitems/edit/18482)

It still narrated "not selected", when I am trying [following example](https://fluentuipr.z22.web.core.windows.net/pull/30687/public-docsite-v9/storybook/index.html?path=/docs/components-tree--default#selection)

probably I am missing something here? please what should be improved with this change? :) 

Thank you for help to understand the issue better. :)

Reply:
just small note, as well I tried NVDA, JAWS with Chome and the narration is the same like before
the VoiceOver with Chrome/Safari the same...

Reply:
as well trying to understand how this  changed in aria 1.2 because I can still see this state as required in doc:
https://www.w3.org/TR/wai-aria-1.2/#treeitem
![image](https://github.com/microsoft/fluentui/assets/22620150/655dba93-21a3-4b61-b357-88eefef20d81)

Reply:
Hi @smhigley 
thanks for explanation and pointing to 1.3 guidance. 

Yes I fully agree we should follow 1.3 guidance.